addon/components/polaris-date-picker.js

```diff
@@ -10,6 +11,8 @@ import {
   getNextDisplayMonth,
   getPreviousDisplayYear,
   getPreviousDisplayMonth,
+  weekdays, // obj
```

Comment:
Is the `// obj` comment needed here?

Reply:
Ah! That was supposed to be a temporary reminder to myself since Polaris uses enums for weekdays haha. Will remove..